lib/Frontend/ParseableInterfaceSupport.cpp

```diff
@@ -279,6 +279,7 @@ static bool buildSwiftModuleFromSwiftInterface(
     StringRef ModuleCachePath, DependencyTracker *OuterTracker) {
   bool SubError = false;
   bool RunSuccess = llvm::CrashRecoveryContext().RunSafelyOnThread([&] {
+    (void)llvm::sys::fs::create_directory(ModuleCachePath);
```

Comment:
Something later will fail. I suppose we could diagnose this directly, since Clang doesn't and it's been a bit of a mystery when that happens, but it seemed like a reasonable starting point.